windows/src/developer/kmcomp/main.pas

```diff
@@ -293,7 +293,7 @@ function CompileKeyboard(FInFile, FOutFile: string; FDebug, FWarnAsError: Boolea
   begin
     with TCompileKeymanWeb.Create do
     try
-      Result := Compile(nil, FInFile, FOutFile, FDebug, @CompilerMessage);   // I3681   // I4865   // I4866
+      Result := Compile(nil, FInFile, FOutFile, FDebug, @CompilerMessageW);   // I3681   // I4865   // I4866
```

Comment:
Does l. 304 keep `@CompilerMessage`?

Reply:
Yes -- kmcmpdll still uses non-Unicode error messages, whereas the Delphi code uses Unicode messages.